src/App.js

```diff
@@ -0,0 +1,25 @@
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Navbar from './components/Navbar/Navbar';
+import store from './Redux/ConfigureStore';
+import Rockets from './pages/Rockets';
+import Missions from './pages/Missions';
+import MyProfile from './pages/MyProfile';
+
+function App() {
+  return (
+    <div className="App">
+      <Provider store={store}>
+        <Navbar />
+        <Routes>
+          <Route path="/Rockets.js" element={<Rockets />} />
+          <Route path="/Missions.js" element={<Missions />} />
+          <Route path="/" element={<MyProfile />} />
```

Comment:
- [ ] You made a great job working on the routing of your project! 💯 However, according to the requirements, Rockets should be the default view! So please consider applying this change. 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/216719392-186db4dc-bf03-4306-b98b-5570882f8a82.png)


- [ ] **[OPTIONAL]:** Also, please avoid including file extensions in the path, they make the path string looks kind of dirty, and they are not needed at all! 🤓 🔧